manuscript/200_What_does_it_mean_to_compose_objects.md

```diff
@@ -14,7 +14,7 @@ Basically it means that an object has obtained a reference to another object and
 If we take the web metaphor a little bit further, we can note some similarities to e.g. a TCP/IP network:
 
 1.  An object can send **messages** to other objects (i.e. call methods on them -- arrows on the above diagram) via **interfaces**. Each message has a **sender** and at least one **recipient**.
-2.  To send a message to a recipient, a sender has to acquire an **address** of the recipient, which, in object-oriented world, we call a reference (actually, in languages such as C++, references are just that -- addresses in memory).
+2.  To send a message to a recipient, a sender has to acquire an **address** of the recipient, which, in object-oriented world, we call a reference (in languages such as C++, references are just that -- addresses in memory).
```

Comment:
Here, when I said:
"(in languages such as C++, references are just that -- addresses in memory)"
I meant something like 
"(and in languages such as C++, references are just that -- addresses in memory)"
Can we change it to add the "and"?